cloudigrade/account/reports.py

```diff
@@ -65,6 +65,9 @@ def get_hourly_usage(start, end, account_id):
         magical combination of Red Hat product version and AWS instance type.
 
     """
+    # Verify that we can get the requested account before proceeding.
+    Account.objects.get(account_id=account_id)
```

Comment:
We should probably use `Account.objects.filter(account_id=account_id).exists()`, it should be a little faster since we are just checking for the presence of the object. Additionally, maybe we could add `if not` and throw an exception if the account does not exist?